website/docs/seo.mdx

```diff
@@ -67,15 +67,13 @@ Similar to [global metadata](#global-metadata), Docusaurus also allows for the a
   <meta name="keywords" content="cooking, blog" />
   <meta name="twitter:card" content="summary_large_image" />
   <link rel="preconnect" href="https://example.com" />
-  <script type="application/ld+json">
-    {JSON.stringify({
-      '@context': 'https://schema.org/',
-      '@type': 'Organization',
-      name: 'Meta Open Source',
-      url: 'https://opensource.fb.com/',
-      logo: 'https://opensource.fb.com/img/logos/Meta-Open-Source.svg',
-    })}
-  </script>
+  <StructuredData structuredData={{
+    '@context': 'https://schema.org/',
+    '@type': 'Organization',
+    name: 'Meta Open Source',
+    url: 'https://opensource.fb.com/',
+    logo: 'https://opensource.fb.com/img/logos/Meta-Open-Source.svg',
+  }} />
```

Comment:
so, it should be rendered in `<head>` finally? 🤪 

ChatGTP seems to suggest rendering it in `<head>`. 
I don't know where it found the information but that seems reasonable to me.

If we use that helper here, we should also show how to import it.

MDX docs automatically support `@docusaurus/Head` through the `<head>` tag but this component requires an import.

Reply:
we show how to import it in the code below - just scroll down! 

Feel free to change this if you would like - the reason I altered it at all was because it used the problematic approach of **not** using `dangerouslySetInnerHTML` ([see comment](https://github.com/facebook/docusaurus/pull/9669#discussion_r1484993080)) that it advocates currently.

We could put this back to a raw `<script>` tag approach but we should switch to `dangerouslySetInnerHTML` so we don't advise people to use code that will upset the Google Search Console.

Reply:
Fixed (I think)